tests/console/dracut_enhanced.pm

```diff
@@ -45,18 +52,19 @@ sub run {
         assert_script_run('for p in *patch; do patch -p0 < $p; done');
         assert_script_run "cd /usr/src/packages/BUILD/dracut-*/test/$dracut_test";
         assert_script_run 'mkdir /tmp/logs';
-        assert_script_run "./test.sh --setup |& tee /tmp/logs/$dracut_test-setup.log", 300;
-        assert_script_run "grep -q dracut-root-block-created /tmp/logs/$dracut_test-setup.log";
+        assert_script_run "./test.sh --setup 2>&1 | tee /tmp/logs/$dracut_test-setup.log", 300;
+        assert_screen("dracut-root-block-success", 180);
         power_action('reboot', textmode => 1);
-        wait_still_screen(10, 60);
+        sleep 10;
```

Comment:
I 'm thinking maybe you could do without the `sleep` since you have the timeout in `assert_screen("linux-login", 600);`